samples/stock/demo/flags-general/demo.js

```diff
@@ -2,62 +2,65 @@
 
     // Load the dataset
     const data = await fetch(
-        'https://www.highcharts.com/samples/data/usdeur.json'
+        'https://cdn.jsdelivr.net/gh/highcharts/highcharts@1485682/samples/data/btc-historical.json'
     ).then(response => response.json());
 
     // Create the chart
     Highcharts.stockChart('container', {
-
-        rangeSelector: {
-            selected: 0
+        accessibility: {
+            typeDescription: `Stock chart with a line series and a flags series
+            indicating key events.`
         },
 
         title: {
-            text: 'USD to EUR exchange rate'
-        },
-
-        tooltip: {
-            style: {
-                width: '200px'
-            },
-            valueDecimals: 4,
-            shared: true
+            text: 'Bitcoin Historical Price and Halvings'
         },
 
-        yAxis: {
-            title: {
-                text: 'Exchange rate'
-            }
+        xAxis: {
+            overscroll: 31536000000 // 1 year
```

Comment:
It seems you added extra space for flag? However, this makes RS buttons a bit useless on initial extremes. What do you think about changing this to one month, then removing 1M button from RS and setting `shape: 'squarepin'` for flags? So the connector is rendered in the middle of flag, not on the left edge